src/components/posts/PostItem.js

```diff
@@ -1,22 +1,24 @@
+import defaultPFP from "../../assets/default.png";
+
 const PostItem = ({ post }) => {
   return (
-    <li className='post-item'>
-      <div className='post-header-wrap'>
-        <div className='post-profile-wrap'>
+    <li className="post-item">
```

Comment:
Seems like you're changing a lot of the quotes in this file to double quotes. Can we keep them to single quotes please